app/org/sagebionetworks/bridge/models/surveys/SurveyRule.java

```diff
@@ -133,12 +173,29 @@ public Builder withEndSurvey(Boolean endSurvey) {
             }
             return this;
         }
+        public Builder withDisplayIf(Boolean displayIf) {
+            if (Boolean.TRUE.equals(displayIf)) {
+                this.displayIf = displayIf;
+            }
```

Comment:
If the user sends a property set to false (which they could do), we set the object field to null so that it is not serialized and sent back in future JSON. Only one boolean property can be true at a time, and this ensures only the true property will be in JSON representations. I've added a test to SurveyRuleTest to verify this.